### PR TITLE
Adngdb 1407623 new source input

### PR DESCRIPTION
**Pull request by mathjazz**
A couple of nits.

### CODE REVIEW

File: pontoon/administration/templates/admin_project_strings.html [hunk 1]
```diff
@@ -9,16 +9,13 @@
 
   <h1>
     {{ 'Strings for %(project)s' | format(project=project.name) }}
-    <aside>
-      <a href="{{ url('pontoon.admin.project.strings', project.slug) }}?format=csv">Download Strings</a>
-    </aside>
```
Comment (adngdb) [inline pontoon/administration/templates/admin_project_strings.html:14]: I instinctively disagree with removing this, but I'm sure you have excellent reasons. Please enlighten me! :smile:

File: pontoon/administration/templates/admin_project.html [hunk 2]
```diff
@@ -78,11 +76,10 @@ <h3>
...
     <div class="clearfix">
-      <textarea name="new_strings" class="strings-source" placeholder="Enter new strings"></textarea>
-      <p>
-        Enter strings to create the first entities. Strings are separated on new line
-        characters. Content must be in en-US locale.
-      </p>
+      <label for="id_new_strings">
+        Enter strings to create the first entities. Strings are separated on new line characters.
+      </label>
+      <textarea id="id_new_strings" name="new_strings" class="strings-source" placeholder="First string"></textarea>
```
Comment (adngdb) [inline pontoon/administration/templates/admin_project.html:82]: `First strings`, pluralized, seems better to me.

**Review by adngdb — COMMENTED** [review]
Thanks a lot for helping me with this!

File: pontoon/administration/templates/admin_project.html (hunk 2 above)
Comment (mathjazz) [reply pontoon/administration/templates/admin_project.html:82]: Agreed.

File: pontoon/administration/templates/admin_project_strings.html (hunk 1 above)
Comment (mathjazz) [reply pontoon/administration/templates/admin_project_strings.html:14]: I don't have better arguments than what the commit message says. :) https://github.com/adngdb/pontoon/pull/1/commits/5db4d5f946e4d750eb289e49b59ae29d756dda51

**Pull request merged**